.snyk

```diff
@@ -7,4 +7,9 @@ ignore:
         reason: >-
           No remediation available yet; Not affecting us since the storage is not accessible to any other client
         expires: 2021-01-10T06:00:00.000Z
+  SNYK-PYTHON-SQLALCHEMY-173678:
+    - '*':
+        reason: >-
+          No remediation path available
```

Comment:
I think we need to include a little more information than this. For example, it _looks_ like there is an [upgrade path available](https://snyk.io/vuln/SNYK-PYTHON-SQLALCHEMY-173678), update to 1.2.18. Can you include why we're not updating (and maybe link to an issue if there is one) and why the risk is acceptable?

For a high severity issue, we should be resolving it within 30 days, so an expiration of 9/1 is too far out.

Reply:
Let me know what you think of this updated version...